lib/ansible/modules/cloud/azure/azure_rm_aks.py

```diff
@@ -77,6 +77,10 @@
             os_disk_size_gb:
                 description:
                     - Size of the OS disk.
+            vnet_subnet_id:
+                description:
+                    - The Subnet ID for a existing Subnet or if you use a different CIDR for the Nodes IP (Kubenet or Azure CNI)
+                    - Note: Does not overlap 169.254.0.0/16, 172.30.0.0/16, 172.31.0.0/16, 192.0.2.0/24
```

Comment:
I think removing the `:` will fix the CI failure. However, I find this line confusing too. Are the ranges shown here examples of what not to do? What to do? Could we remove them altogether?

```suggestion
                    - Cannot overlap with other address ranges within the same virtual network.
```